src/components/Dashboard/Dashboard.js

```diff
@@ -74,18 +73,12 @@ const card = (pathway, userEmail, subscribed, state) => {
 const getAwarded = async(email) =>{
     var resp = null;
 
-    await axios({
-        method: 'get',
-        url: `/award/user`,
-        data: {
-          body: {
-            email: email
-          }
-        }
-    })
+    await axios
+        .get(
+            `/award`
+        )
     .then(res => {
```

Comment:
Let's try to avoid using the promise callbacks. Prefer the async and await syntax.